File: app/src/main/java/com/kidozh/discuzhub/results/LoginResult.java
package com.kidozh.discuzhub.results;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class LoginResult extends BaseResult {
    @JsonIgnoreProperties(ignoreUnknown = true)
    @JsonProperty("Variables")
    public VariableResults variables;

}

File: app/src/main/java/com/kidozh/discuzhub/results/LoginResult.kt
package com.kidozh.discuzhub.results

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class LoginResult : BaseResult() {
    @JsonIgnoreProperties(ignoreUnknown = true)
    @JsonProperty("Variables")
    var variables: LoginVariableResult = LoginVariableResult()

    class LoginVariableResult : VariableResults() {
        var loginUrl: String? = null
    }
}